app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -56,47 +65,51 @@
+                    android:layout_marginTop="16dp"
                     android:fontFamily="@font/epilogue_bold"
                     android:text="@string/sign_up"
-                    android:textColor="#379AE6"
-                    android:textSize="24sp"
+                    android:textColor="@color/blue"
+                    android:textSize="28sp"
                     app:layout_constraintStart_toStartOf="@+id/reference"
-                    app:layout_constraintTop_toTopOf="parent"
-                    android:layout_marginTop="20dp"/>
+                    app:layout_constraintTop_toTopOf="parent" />
 
                 <TextView
                     android:id="@+id/tvCreateAccount"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="10dp"
+                    android:layout_marginTop="12dp"
                     android:fontFamily="@font/epilogue_semibold"
                     android:text="@string/create_an_account"
-                    android:textColor="#9B9B9B"
-                    android:textSize="16sp"
+                    android:textColor="@color/light_grey"
+                    android:textSize="20sp"
                     app:layout_constraintStart_toStartOf="@+id/tvSignUp"
                     app:layout_constraintTop_toBottomOf="@+id/tvSignUp" />
 
                 <TextView
                     android:id="@+id/tvName"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="44dp"
+                    android:layout_marginTop="32dp"
                     android:text="@string/name"
-                    android:textColor="#323842"
-                    android:textSize="15sp"
-                    app:layout_constraintStart_toStartOf="@+id/tvCreateAccount"
+                    android:fontFamily="@font/epilogue_medium"
+                    android:textColor="@color/light_black"
+                    android:textSize="20sp"
+                    app:layout_constraintStart_toStartOf="@+id/etName"
                     app:layout_constraintTop_toBottomOf="@+id/tvCreateAccount" />
 
                 <EditText
                     android:id="@+id/etName"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:layout_marginHorizontal="36dp"
                     android:layout_marginTop="12dp"
-                    android:layout_marginHorizontal="50dp"
-                    android:padding="5dp"
                     android:background="@drawable/custom_edittext"
-                    android:elevation="5dp"
+                    android:elevation="4dp"
+                    android:hint="@string/hint_name"
+                    android:fontFamily="@font/inter_regular"
+                    android:padding="4dp"
```

Comment:
Increase the padding to 12dp for all Edit Text Views. Ensure that the layout fits in the screen, if it doesn't reduce padding accordingly